SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/Channel/IChannel.cs

```diff
@@ -52,5 +52,11 @@ public interface IChannel : IDisposable
         /// </summary>
         /// <param name="isAllowed">True if network request allowed, false otherwise.</param>
         void SetNetworkRequestAllowed(bool isAllowed);
+
+        /// <summary>
+        /// Check if there are any pending logs in database.
+        /// Trigger ingestion if logs are found.
```

Comment:
/// <summary>
/// Check if there are any pending logs in database and trigger ingestion if logs are found.
/// </summary>